src/components/Center/podcast.js

```diff
@@ -1,4 +1,4 @@
-import { ListOfPodcast } from "./listOfPodcast";
+import { PlaylistSuggestion } from "./playlistSuggestion";
```

Comment:
That still looks almost the same as `dailyMix` file. Create one and reuse this with different props (title, type to filter)